spread.yaml

```diff
@@ -335,41 +335,53 @@ backends:
             ADDRESS $SPREAD_EXTERNAL_ADDRESS
         systems:
             - ubuntu-core-16-64:
-                  username: external
-                  password: ubuntu
+                  username: '$(HOST: echo "${SPREAD_SSH_USER:-external}")'
+                  password: '$(HOST: echo "${SPREAD_SSH_PASSWORD:-ubuntu}")'
+                  ssh-rsa-key: '$(HOST: echo "${SPREAD_SSH_RSA_KEY:-}")'
```

Comment:
Nice!

Could we move all of those above, under the external backend? Not sure if spread allows this kind of _inheritance_ but it would help with readability a lot